src/client/proxy/http.cc

```diff
@@ -220,47 +274,71 @@ void HTTPProxyHandler::HandleStreamRequestComplete(
           stream);
     GetOwner()->AddHandler(connection);
     connection->I2PConnect(
-        reinterpret_cast<const std::uint8_t*>(m_Request.data()),
-        m_Request.size());
+        reinterpret_cast<const uint8_t*>(m_Protocol.m_Request.c_str()),
```

Comment:
Please use `std::size_t` and the standard library for all types when possible. Calling `size_t` directly is deprecated and the only reason this compiles now is because of deprecations elsewhere that have included `stddef.h`.